web/src/beta/features/Navbar/LeftSection/Profile/index.tsx

```diff
@@ -92,98 +32,52 @@ const HeaderProfile: React.FC<Props> = ({
+      subItem: workspaces.map(w => {
+        return {
+          id: w.id as string,
+          title: w.name ?? t("Unknown"),
+          hasCustomSubMenu: true,
+          personal: w.personal,
+          selected: currentWorkspace.id === w.id,
+          onClick: () => w.id && handleWorkspaceChange(w.id),
+        };
+      }),
+      title: t("Switch workspace"),
+    },
     {
-      text: t("Workspaces"),
-      items: [
-        ...workspaces.map(w => {
-          return {
-            text: w.name ?? t("Unknown"),
-            selected: currentWorkspace.id === w.id,
-            onClick: () => w.id && handleWorkspaceChange(w.id),
-          };
-        }),
-        { text: t("New Workspace"), icon: "workspaceAdd", onClick: openModal },
-      ],
+      icon: "exit",
+      id: "logOut",
+      onClick: onSignOut,
+      title: t("Log out"),
+    },
+    {
+      icon: "file",
+      id: "assetManagement",
+      onClick: handleAssetManager,
+      title: t("Asset management"),
     },
-    { text: t("Log out"), onClick: onSignOut },
-    { breakpoint: true },
-    { text: `v${__APP_VERSION__}` },
   ];
 
-  return <Menu label={currentWorkspace.name} items={menuItems} />;
+  return (
+    <Option size="body">
+      <PopupMenu label={currentWorkspace.name} menu={popupMenu} />
+    </Option>
+  );
 };
 
-const InputWrapper = styled.div`
-  display: flex;
-  position: relative;
-  cursor: pointer;
-  align-items: center;
-`;
-
-const Label = styled(Text)<{ open: boolean }>`
-  padding: 7px 12px;
-  /* The width + placement of the arrow icon */
-  padding-right: 30px;
-  width: 100%;
-  border-radius: 4px;
-  color: ${({ theme }) => theme.content.weak};
-
-  &:focus {
-    border: 1px solid ${({ theme }) => theme.select.strong};
-  }
-
-  &:focus-visible {
-    border: 1px solid ${({ theme }) => theme.select.strong};
-    outline: none;
-  }
-
-  &:hover {
-    color: ${({ theme }) => theme.content.main};
-    background: ${({ theme }) => theme.bg[2]};
-  }
-`;
-
-const ArrowIcon = styled(Icon)<{ open: boolean }>`
-  position: absolute;
-  right: 10px;
-  top: 50%;
-  transform: ${({ open }) => (open ? "translateY(-50%) scaleY(-1)" : "translateY(-50%)")};
-  color: ${({ theme }) => theme.content.weak};
-`;
-
-const PickerWrapper = styled(Popover.Content)`
-  min-width: 200px;
-  outline: none;
-  border-radius: 4px;
-  background: ${({ theme }) => theme.bg[0]};
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-`;
-
-const Spacer = styled.div`
-  border-top: 0.5px solid ${({ theme }) => theme.outline.weak};
-  margin: 2px 0;
-`;
-
-const StyledLinkButton = styled(Link)`
-  text-decoration: none;
-  padding: 9px 12px;
-  font-size: 14px;
-  border-radius: 4px;
-  display: flex;
-  align-items: center;
-  gap: 5px;
-  color: ${({ theme }) => theme.content.main};
-  :hover {
-    text-decoration: none;
-    background: ${({ theme }) => theme.bg[2]};
-  }
-`;
-
 const Option = styled(Text)`
```

Comment:
please don't use any component from `beta/components` insteady you can use `Typograph` from `beta/lib/reeath-ui/components`

Reply:
I think it's updated.